config.yaml

```diff
@@ -54,7 +54,9 @@ build_iso:
             - "open-power-host-os-virt-management"
         OpenPOWER Host OS RAS:
             - "open-power-host-os-ras"
-    iso_root_fs_packages_groups::
+    minimal_install_groups:
+        - "core"
```

Comment:
- OK
- Splitted lists for groups and single packages (`grub2`) below to remove the confusing `@`s.